.env.example

```
@@ -1 +1,2 @@
-MY_SECRET="MY_SECRET"
+APP_ID=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)